plugins/cross-links.js

```diff
@@ -1,293 +1,268 @@
 // @author         mcben
 // @name           Cross links
 // @category       Draw
-// @version        1.2.0
+// @version        1.3.0
 // @description    Checks for existing links that cross planned links. Requires draw-tools plugin.
 
 
-window.plugin.crossLinks = function() {};
+window.plugin.crossLinks = function () { };
 
-window.plugin.crossLinks.greatCircleArcIntersect = function(a0,a1,b0,b1) {
-  // based on the formula at http://williams.best.vwh.net/avform.htm#Int
+/**
+ * greatCircleArcIntersect
+ */
+window.plugin.crossLinks.greatCircleArcIntersect = function (a0, a1, b0, b1) {
```

Comment:
If this (and others) function can be useful for 3rd-party plugins, then better to add short description of it's action and arguments.